server/lib/config/pbench-server-default.cfg

```diff
@@ -122,6 +122,12 @@ webserver = %(default-host)s
 host-info-path-prefix = pbench-results-host-info.versioned/pbench-results-host-info.URL
 host_info_url = http://%(webserver)s/%(host-info-path-prefix)s
 
+# hostport logger type uses UDP-based logging.
```

Comment:
Can you move this comment to the line before `logger_host`?

Reply:
sure